raiden-ts/tests/unit/epics/transport.spec.ts

```diff
@@ -1828,3 +1836,1716 @@ describe('transport epic', () => {
     });
   });
 });
+
+describe('transport epic', () => {
+  const matrixServer = 'matrix.raiden.test'; // define later in fixture constant
+  const partnerRoomId = `!partnerRoomId:${matrixServer}`;
+  const processed: Processed = {
+    type: MessageType.PROCESSED,
+    message_identifier: makeMessageId(),
+  };
+  const fetch = jest.fn(async () => ({
+    ok: true,
+    status: 200,
+    text: jest.fn(async () => `- ${matrixServer}`),
+  }));
+  Object.assign(global, { fetch });
+
+  describe('initMatrixEpic', () => {
+    test('matrix stored setup', async () => {
+      expect.assertions(2);
+      const [raiden] = await makeRaidens(1);
```

Comment:
You should use `makeRaiden()` when creating a single instance. `makeRaidens` is more to make a batch of mocked instances.